### ISSUE TITLE
/graphql responding with 503 after 4 min

### ISSUE THREAD

**Issue by lorensr** [apollographql/meteor-integration#69]
Just released `meteor/apollo@0.3.0`, but I think there's a bug in it. When I pointed `meteor-starter-kit` at `meteor-integration` locally, it worked, but after releasing and pointing at the atmosphere version, I started getting the bad behavior. It continues even when I switch back to local.

![image](https://cloud.githubusercontent.com/assets/251288/22763881/5223c310-ee35-11e6-855a-a98bd9c6443d.png)

![image](https://cloud.githubusercontent.com/assets/251288/22763889/5ca9ebd4-ee35-11e6-9fa9-3e6303c867c6.png)

```
git clone git@github.com:apollographql/meteor-starter-kit.git
git checkout 0.3.0-wip
npm install
meteor
```

**Comment by lorensr** [comment]
Looks like fixed by `rm -rf node_modules` inside of `packages/meteor-integration`. Will try republishing

**Comment by lorensr** [comment]
Appears to be fixed in `0.3.1`. 

@rdickert reminds me of that one time! Gotta delete `node_modules` more often! 

@xavcz would you mind checking this to make sure I'm not crazy?

```sh
git clone git@github.com:apollographql/meteor-starter-kit.git
npm i
meteor
```

http://localhost:3000/graphiql

**Comment by xavcz** [comment]
All good for me! 👌
